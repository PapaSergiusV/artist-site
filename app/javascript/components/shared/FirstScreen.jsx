import React, { Component } from "react";
import CSSModules from "react-css-modules";

import styles from "./FirstScreen.module.scss";

class FirstScreen extends Component {
  render() {
    return (
      <div styleName="wrapper">
        <span>LOGO</span>
        <span>X</span>
      </div>
    );
  }
}

export default CSSModules(FirstScreen, styles, { allowMultiple: true });
// export default FirstScreen;