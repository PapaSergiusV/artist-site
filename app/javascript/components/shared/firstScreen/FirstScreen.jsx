import React from "react";
import CSSModules from "react-css-modules";
import { FaBars } from "react-icons/fa";

import styles from "./FirstScreen.module.scss";

const FirstScreen = props => {
  return (
    <div styleName="wrapper">
      <span>LOGO</span>
      <FaBars onClick={props.openMenu} />
      <div styleName="scroll-button">
        <span>explore</span>
      </div>
    </div>
  );
};

export default CSSModules(FirstScreen, styles, { allowMultiple: true });
// export default FirstScreen;