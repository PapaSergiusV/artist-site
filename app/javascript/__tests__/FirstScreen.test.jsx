import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import "../setupTests";

import FirstScreen from "../components/shared/firstScreen/FirstScreen";

describe("FirstScreen component", () => {
  it("should render logo and menu button", () => {
    const wrapper = shallow(<FirstScreen/>);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
    expect(wrapper.children().length).toEqual(3);
  });
});