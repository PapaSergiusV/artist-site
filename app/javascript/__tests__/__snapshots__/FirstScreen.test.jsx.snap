// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FirstScreen component should render logo and menu button 1`] = `
<div
  className="wrapper"
>
  <span>
    LOGO
  </span>
  <FaBars />
  <div
    className="scroll-button"
  >
    <span>
      explore
    </span>
  </div>
</div>
`;
